modules/references/client/components/create-reference/Feedback.js

```diff
@@ -0,0 +1,42 @@
+import PropTypes from 'prop-types';
+import React from 'react';
+import { useTranslation } from 'react-i18next';
+
+export default function Feedback({ feedback, onChangeFeedback }) {
+  const { t } = useTranslation('references');
+
+  return (
+    <div className="panel panel-default">
+      <div className="panel-heading">
+        <h4 id="feedback-public-question">
+          {t('Would you like to describe something about them?')}
+        </h4>
+      </div>
+      <div
+        className="panel-body"
+        role="group"
+        aria-labelledby="feedback-public-question"
+      >
+        <label htmlFor="feedback-message" className="control-label">
+          {t('Public feedback')}
+        </label>
+        <textarea
+          className="form-control input-lg"
+          rows="7"
+          id="feedback-message"
+          onChange={event => onChangeFeedback(event.target.value)}
+          value={feedback}
+        ></textarea>
+        <span className="help-block">
+          {t('Were they great at cooking? singing? dancing?')}
```

Comment:
Non-blocking:

I think this guideline would encourage people to judge.
I would prefer a culture in which people say `I felt such and such, experienced that and that, needed that and that.` instead of `This person was such and such.`. This point of view is shaped heavily by my understanding of [Nonviolent Communication](https://en.wikipedia.org/wiki/Nonviolent_Communication).
This should not block merging. If there is a desire to talk about this, let's do it independently of this PR; and iterate on the guideline only after such discussion.
I just really wanted to express my point of view on this, because it matters to me.